store/store_test.go

```diff
@@ -213,6 +213,16 @@ func testDataForLatestTests() map[string]bson.M {
 	return testData
 }
 
+func outputData(inputData bson.M) bson.M {
```

Comment:
Consider calling this `dropInternalKeys` or `dropInternalKeys` to improve readability